.github/workflows/release.yml

```diff
@@ -2,92 +2,39 @@ name: Release
-          node-version-file: '.node-version'
-
-      - name: Set git settings
-        uses: fregante/setup-git-user@2e28d51939d2a84005a917d2f844090637f435f8 # commit hash = v1.1.0
+          node-version: '18' # This workflow ignores node-version-file because semantic-release requires node 18 and cloudflare-wrangler-action requires node 16
 
       - name: Install Node packages
         run: yarn install
-
-      - name: Disable version tags
-        run: yarn config set version-git-tag false
-      - name: Bump version
-        run: yarn version --${{ github.event.inputs.version_bump }}
-      - name: Build
-        run: yarn build
       - name: Lint
         run: yarn lint
       - name: Tests
         run: yarn test
-
+      - name: Build
+        run: yarn build
       - name: Type check
         run: yarn test:dts
-
-      - name: Get version
-        id: version
-        uses: notiz-dev/github-action-json-property@a5a9c668b16513c737c3e1f8956772c99c73f6e8 # commit hash = v0.2.0
-        with:
-          path: 'package.json'
-          prop_path: 'version'
-
-      - name: Commit and push a new version of the scripts
-        run: |
-          git commit package.json dist/ -m "build: new scripts v${{steps.version.outputs.prop}}"
-          git push "https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${GITHUB_REPOSITORY}.git"
-          git tag -a "v${{steps.version.outputs.prop}}" -m "v${{steps.version.outputs.prop}}"
-          git push --tags "https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${GITHUB_REPOSITORY}.git"
-
-      - name: Create a GitHub release
-        uses: ncipollo/release-action@18990f60e0df5c7b33d19565c84c7505b326aeca # commit hash = v1.10.0
-        with:
-          tag: v${{steps.version.outputs.prop}}
-          name: Release ${{steps.version.outputs.prop}}
-          artifacts: "dist/fingerprintjs-pro-cloudflare-worker.d.ts,dist/fingerprintjs-pro-cloudflare-worker.esm.js"
-#       Comment out below part because PRs created by bots don't trigger GitHub checks and the PR gets stuck
-#       More info: https://github.com/peter-evans/create-pull-request/issues/48
-#      - name: Create Pull Request from develop to main
-#        id: create_pr
-#        uses: devops-infra/action-pull-request@a671d0a772ef71131f970e6f02f1e11334f99eab # commit hash = v0.5.0
-#        with:
-#          github_token: ${{ secrets.GITHUB_TOKEN }}
-#          source_branch: develop
-#          target_branch: main
-#          title: New version of scripts v${{ steps.version.outputs.prop }}
-#          body: New version of scripts v${{ steps.version.outputs.prop }}
-#
-#      - name: Check PR outputs
-#        if: ${{ steps.create_pr.outputs.url }}
-#        run: echo "Pull Request URL - ${{ steps.create_pr.outputs.url }}"
+      - name: Install release dependencies
+        run: yarn install --cwd=release
+      - name: Release
+        run: ./release/node_modules/.bin/semantic-release --extends ./release/.releaserc.js
```

Comment:
Why not `yarn release` instead? @Orkuncakilkaya

Reply:
No specific reason, that's the old approach we used on SDK projects and there was no need to change. So when I used SDK projects' workflow as a template this just came as it is